File: 1024hw/hw16.c
#include <stdio.h>
#include <windows.h>
int main() {
	int age;
	printf("Enter your age : ");
	scanf_s("%d",&age);
	printf("%s", (age > 16 ? "You are enough old to drive car." : "You are not old enough to drive car"));

	system("pause");
}

File: 1024java/hw16.java
import java.util.Scanner;

public class hw16 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("How old are you ? ");
        int age = scan.nextInt();
        if(age >= 16){
            System.out.print("You are old enough to drive");
        }
        else{
            System.out.print("You can't drive wait for "+ (16-age) +" year");
        }
    }

}
